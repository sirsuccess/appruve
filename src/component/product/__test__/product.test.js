import React from "react";
import renderer from "react-test-renderer";
import DummyData from "../../../dummyData/cardData";

import Component from "../product";

it("renders correctly when called ", () => {
  const tree = renderer
    .create(<Component product={DummyData[0].product} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
