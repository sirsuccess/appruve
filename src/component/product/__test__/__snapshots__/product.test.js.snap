// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when called  1`] = `
Array [
  <div
    className="section12-line"
  >
    Listening Mock Tests
  </div>,
  <div
    className="section12-line"
  >
    Listening Mock Tests
  </div>,
  <div
    className="section12-line"
  >
    Reading Mock Tests
  </div>,
  <div
    className="section12-line"
  >
    
  </div>,
  <div
    className="section12-line"
  >
    
  </div>,
  <div
    className="section12-line"
  >
    
  </div>,
  <div
    className="section12-line"
  >
    
  </div>,
]
`;
